docs/data/date-pickers/date-picker/date-picker.md

```diff
@@ -42,27 +35,34 @@ Check-out their documentation page for more information:
 
 ## Uncontrolled vs. Controlled
 
-The component can be uncontrolled or controlled
+The Date Picker component can be uncontrolled or controlled
 
 {{"demo": "DatePickerValue.js"}}
 
 ## Responsiveness
 
-The Date Picker component is designed and optimized for the device it runs on.
+The Date Picker component is available in three variants:
```

Comment:
I tried to combine these two paragraph
I'll wait for feedbacks before applying the same changes on the other pages

Reply:
IMHO it looks more cohesive as we have all 4 component variations in one place. Maybe one final small nitpick could be that the `Responsiveness` section title is probably not the most correct anymore. 🙈

Reply:
Maybe one of the following would work better:
- `Component variants`
- `Component variations`
- `Component versions`
- `Available components`
- `Available variants`

Reply:
We would loose the "responsive" part which is very explicit for people looking for the mobile / desktop only variants (no idea how many people actually want those components and not the responsive one).

`Desktop / Mobile and Static variants` is a little verbose but very explicit.

Otherwise, `Available components` is fine for me